decrypt/decrypt.go

```diff
@@ -69,37 +65,38 @@ func Decrypt(args []string) error {
 		return errors.New("a private key is required to decrypt data")
 	}
 
-	var privateKey *[32]byte
-
-	// try reading private key without password
-	privateKey, err = readPrivateKeyFile(*privateKeyFile, "")
-	if err != nil {
-
-		// if there was an error, try again with the password
-		password, err := getPassword("C4GH_PASSWORD")
-		if err != nil {
-			return err
-		}
-
-		// Loading private key file
-		privateKey, err = readPrivateKeyFile(*privateKeyFile, password)
+	password, available := os.LookupEnv("C4GH_PASSWORD")
+	if !available {
+		password, err = helpers.PromptPassword("Enter password to unlock private key")
```

Comment:
This makes the tool less user friendly imo.

Reply:
As long as the user password protects the private key there is no difference in the behavior of the tool.